### ISSUE TITLE
Waymo data different with data in process_waymo_script.py

### ISSUE THREAD

**Issue by LiuXuan66** [yifanlu0227/ChatSim#6]
![截图 2024-03-22 16-05-33](https://github.com/yifanlu0227/ChatSim/assets/61314087/17955229-fec9-4943-b757-094ae268513c)

below is process_waymo_script:
![截图 2024-03-22 16-07-54](https://github.com/yifanlu0227/ChatSim/assets/61314087/10fbc111-8621-490f-a6a0-4e3c37f3da7f)
I download the waymo dataset as your readme:v1.4.2/individual_files/training, but they are different with the scene names in process_waymo_scrip.py, so I can't execute the command to get correct data.

**Comment by yifanlu0227** [comment]
Hi, we selected static scenes in the Waymo dataset as shown in `process_waymo_script`. 

You just need to download some of them and then continue.

**Comment by LiuXuan66** [comment]
> Hi, we selected static scenes in the Waymo dataset as shown in `process_waymo_script`.
> 
> You just need to download some of them and then continue.

So could you please offer the paths of these scenes in waymo google cloud, I can't find some scenes in training or val set. And are you sure the data is in waymo v1.4.2 individual files not in other version?

**Comment by ZiWang1105** [comment]
Sorry for not clearly explaining how to download the data. Yes, we use Waymo v1.4.2 individual files.

'segment-10247954040621004675_2180_000_2200_000_with_camera_labels.tfrecord'
'segment-13469905891836363794_4429_660_4449_660_with_camera_labels.tfrecord'
'segment-14333744981238305769_5658_260_5678_260_with_camera_labels.tfrecord'

These three tfrecord files are from the validation set. And the rest of the list are from the training set. 

Maybe you can try to use `gsutil` to download the data like this:

`gsutil cp -n gs://waymo_open_dataset_v_1_4_2/individual_files/validation/segment-10247954040621004675_2180_000_2200_000_with_camera_labels.tfrecord ${your_dest_file_path}`

In addition, if you want to download the files from their website directly, you can use the `filter` function to search the corresponding tfrecord file names and download them.

**Comment by LiuXuan66** [comment]
Thanks for you reply. I'm new to Waymo, and I thought there were 50 files in training because only 50 files are shown in the first page, and now I see the bottom of page that there are 798 files.

**Comment by yifanlu0227** [comment]
Yes, the whole Waymo dataset includes about 1000 sequences (train + validation). Most of them have dynamic objects and we only select those static scenes.

**Comment by LiuXuan66** [comment]
> Yes, the whole Waymo dataset includes about 1000 sequences (train + validation). Most of them have dynamic objects and we only select those static scenes.

Got it! Thank you very much!

**Comment by LiuXuan66** [comment]
Got it! Thank you very much!
